### DISCUSSION TITLE
Feed use many cache not works

### DISCUSSION THREAD

**Discussion by renanbastos93** [community/community#119862] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

For me many times. feed bug not update contents

**Comment by LiteBrite82** [comment]
This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.
